source/jormungandr/tests/vehicle_occupancy_tests.py

```diff
@@ -102,6 +102,8 @@ def _call_webservice(self, arguments):
             }
         if stop_id == "stopP2" and vehicle_journey_id == 'vehicle_journey:vjP:1:modified:0:bib':
             json = {"vehicle_occupancies": []}
+        if stop_id == "stopQ2":
+            return MagicMock(return_value=None)
```

Comment:
Can you test the forseti 503 return code?

Reply:
I tried to test with status_code 503.